lib/elements/elements.dart

```diff
@@ -415,12 +415,12 @@ class TabElement extends DElement {
   final String? name;
   final Function? onSelect;
 
-  TabElement(Element element, {this.name, this.onSelect}) : super(element);
+  TabElement(super.element, {this.name, this.onSelect});
 
   void handleSelected() {
-    if (onSelect != null) onSelect!();
+    onSelect?.call();
   }
 
   @override
-  String toString() => name!;
+  String toString() => '$name';
```

Comment:
Does this convert an exception to `"null"`?

Reply:
Yes it does. I switched to this to match the original functionality of the code, but after a second look, these fields don't need to be nullable anyway. I've adjusted the class in the latest commit :)